packages/path_provider/path_provider/README.md

```diff
@@ -2,28 +2,42 @@
 
 [![pub package](https://img.shields.io/pub/v/path_provider.svg)](https://pub.dev/packages/path_provider)
 
-A Flutter plugin for finding commonly used locations on the filesystem. Supports Android, iOS, Linux, macOS and Windows.
+A Flutter plugin for finding commonly used locations on the filesystem. 
+Supports Android, iOS, Linux, macOS and Windows.
 Not all methods are supported on all platforms.
 
 ## Usage
 
 To use this plugin, add `path_provider` as a [dependency in your pubspec.yaml file](https://flutter.dev/docs/development/platform-integration/platform-channels).
 
-### Example
-
-``` dart
+## Example
+```dart
 Directory tempDir = await getTemporaryDirectory();
 String tempPath = tempDir.path;
 
 Directory appDocDir = await getApplicationDocumentsDirectory();
 String appDocPath = appDocDir.path;
 ```
 
-Please see the example app of this plugin for a full example.
+## Supported platforms and paths
+
+Directories support by platform:
 
-### Usage in tests
+| Directory | Android | iOS | Linux | macOS | Windows |
+| :--- | :---: | :---: | :---: | :---: | :---: |
+| Temporary | ✔️ | ✔️ | ✔️ | ✔️ | ✔️ |
+| Application Support | ✔️ | ✔️ | ✔️ | ✔️ | ✔️ |
+| Application Library | ❌️ | ✔️ | ✔️ | ✔️ | ✔️ |
```

Comment:
I missed this before. This is not supported on Linux or Windows.

Reply:
Fixed.